utils/recog_wav.sh

```diff
@@ -54,26 +71,45 @@ decode_cmd=
 wav=$1
 download_dir=${decode_dir}/download
 
-if [ $# -gt 1 ]; then
-    echo $help_message
+if [ $# -lt 1 ]; then
+    echo "${help_message}"
     exit 1;
 fi
 
 set -e
 set -u
 set -o pipefail
 
+# Check model name or model file is set
+if [ -z $models ]; then
+    if [[ -z $cmvn || -z $lang_model || -z $recog_model || -z $decode_config ]]; then
+    echo 'Error: models or set of cmvn, lang_model, recog_model and decode_config are not set.' >&2
+    exit 1
+    fi
+fi
+
+# Check jq is installed
+if ! [ -x "$(command -v jq)" ]; then
+    echo 'Error: jq is not installed.' >&2
```

Comment:
`jq` is not a standard tool, and it would be better to include the installation of it as an extra tool. (I used to include it in our Makefile, but now I removed it).

Reply:
I see. I will include `jq` installation in Makefile.